assets/app.js

```diff
@@ -0,0 +1,3 @@
+import './styles/app.css';
+
+console.log('Hello Webpack Encore! Edit me in assets/app.js');
```

Comment:
```suggestion
console.log('Hello Webpack Encore! Edit me in assets/app.js');
```
I'm not sure we need this line.